testdata/baselines/reference/submoduleAccepted/compiler/classFieldSuperAccessibleJs1.errors.txt.diff

```diff
@@ -5,27 +5,18 @@
 -
 -
 -==== index.js (1 errors) ====
-+index.js(4,3): error TS2551: Property 'blah2' does not exist on type 'typeof C'. Did you mean 'blah1'?
-+index.js(9,23): error TS2551: Property 'blah2' does not exist on type 'typeof C'. Did you mean 'blah1'?
-+
-+
-+==== index.js (2 errors) ====
-     class C {
-       static blah1 = 123;
-     }
-     C.blah2 = 456;
-+      ~~~~~
-+!!! error TS2551: Property 'blah2' does not exist on type 'typeof C'. Did you mean 'blah1'?
-+!!! related TS2728 index.js:2:10: 'blah1' is declared here.
-     
-     class D extends C {
-       static {
-         console.log(super.blah1);
-         console.log(super.blah2);
-                           ~~~~~
+-    class C {
+-      static blah1 = 123;
+-    }
+-    C.blah2 = 456;
+-    
+-    class D extends C {
+-      static {
+-        console.log(super.blah1);
+-        console.log(super.blah2);
+-                          ~~~~~
 -!!! error TS2565: Property 'blah2' is used before being assigned.
```

Comment:
This is an interesting one; is the reparser creating nodes in a fashion that means that we no longer detect use-before-def? E.g. if I declare a prop on a class, it's technically "declared" and wouldn't error.

Reply:
I think there was special-case code for class expandos before that has changed just enough to not trigger. I'll check on it during my burndown of diffs.